docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -158,11 +158,17 @@ For more information, see the link:https://github.com/zchee/docker-machine-drive
 To use Minishift with Hyper-V:
 
 . Install link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[Hyper-V].
+
+. Add the user to the local Hyper-V Administrators group.
++
+NOTE: These permissions are needed to allow the user to create and delete virtual machines with the Hyper-V Management API. 
```

Comment:
s/These permissions/This is/. As you are trying to explain why user need to do the previous step.

Reply:
This is required to allow the user to create and delete virtual machines with the Hyper-V Management API.